### ISSUE TITLE
Error when verify.js throw errors

### ISSUE THREAD

**Issue by LaplancheMaxime** [droidsolutions/semantic-release-nuget#199]
Hello, 

I have a problem, the errors that throw in the file "src/verify.ts" do not go up in the console. 
I encounter an error, here is the console output:
```
[2:20:54 PM] [semantic-release] › ✖  An error occurred while running semantic-release: AggregateError
    at verify (/builds/mlaplanche/**/node_modules/@droidsolutions-oss/semantic-release-nuget/dist/src/verify.js:50:15)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async validator (/builds/mlaplanche/**/node_modules/semantic-release/lib/plugins/normalize.js:34:24)
    at async /builds/mlaplanche/**/node_modules/semantic-release/lib/plugins/pipeline.js:37:34
    at async /builds/mlaplanche/**/node_modules/semantic-release/lib/plugins/pipeline.js:31:3
    at async Object.pluginsConf.<computed> [as verifyConditions] (/builds/mlaplanche/**/node_modules/semantic-release/lib/plugins/index.js:80:11)
    at async run (/builds/mlaplanche/**/node_modules/semantic-release/index.js:103:3)
    at async module.exports (/builds/mlaplanche/**/node_modules/semantic-release/index.js:269:22)
    at async module.exports (/builds/mlaplanche/**/node_modules/semantic-release/cli.js:55:5) {
  pluginName: '@droidsolutions-oss/semantic-release-nuget'
}
AggregateError: 
    AggregateError
        at verify (/builds/mlaplanche/**/node_modules/@droidsolutions-oss/semantic-release-nuget/dist/src/verify.js:50:15)
        at processTicksAndRejections (node:internal/process/task_queues:96:5)
        at async validator (/builds/mlaplanche/**/node_modules/semantic-release/lib/plugins/normalize.js:34:24)
        at async /builds/mlaplanche/**/node_modules/semantic-release/lib/plugins/pipeline.js:37:34
        at async /builds/mlaplanche/**/node_modules/semantic-release/lib/plugins/pipeline.js:31:3
        at async Object.pluginsConf.<computed> [as verifyConditions] (/builds/mlaplanche/**/node_modules/semantic-release/lib/plugins/index.js:80:11)
        at async run (/builds/mlaplanche/**/node_modules/semantic-release/index.js:103:3)
        at async module.exports (/builds/mlaplanche/**/node_modules/semantic-release/index.js:269:22)
        at async module.exports (/builds/mlaplanche/**/node_modules/semantic-release/cli.js:55:5)
    at /builds/mlaplanche/**/node_modules/semantic-release/lib/plugins/pipeline.js:54:11
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
    at async Object.pluginsConf.<computed> [as verifyConditions] (/builds/mlaplanche/**/node_modules/semantic-release/lib/plugins/index.js:80:11)
    at async run (/builds/mlaplanche/**/node_modules/semantic-release/index.js:103:3)
    at async module.exports (/builds/mlaplanche/**/node_modules/semantic-release/index.js:269:22)
    at async module.exports (/builds/mlaplanche/**/node_modules/semantic-release/cli.js:55:5)
```
I use the latest version.

Thank you for your feedback

**Comment by Kampfmoehre** [comment]
Hi,

can you test this version of the plugin:
https://www.npmjs.com/package/@droidsolutions-oss/semantic-release-nuget/v/1.1.2-beta.1

**Comment by Kampfmoehre** [comment]
@LaplancheMaxime Could you verify that the error message is visible now? I would like like to merge it but hadn't had an opportunity to test it myself.

**Comment by Kampfmoehre** [comment]
I merged the changes to the main branch which is included in [1.2.0](https://github.com/droidsolutions/semantic-release-nuget/releases/tag/v1.2.0). I'll close this issue, if the problem containues to exist, please comment and I'll reopen it.
